.github/workflows/npm-publish.yml

```
@@ -21,12 +21,12 @@ jobs:
         with:
           app_id: ${{ vars.FUNDABOT_APP_ID }}
           private_key: ${{ secrets.FUNDABOT_PRIVATE_KEY }}
-      
+
       - uses: actions/setup-node@v3
```

actionlint (1.7.8)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)